examples/edge-functions/README.md

```diff
@@ -0,0 +1,40 @@
+# Supabase Edge Function Examples
+
+## What are Supabase Edge Functions?
+
+[Supabase Edge Functions](https://supabase.com/edge-functions) are written in TypeScript, run via Deno, and deployed with the Supabase CLI. Please [download](https://github.com/supabase/cli#install-the-cli) the latest version of the Supabase CLI, or [upgrade](https://github.com/supabase/cli#install-the-cli) it if you have it already installed.
+
+## Example Functions
+
+The function examples are located in [`./supabase/functions`](./supabase/functions):
+
+- Retrieve data from an authenticated user via RLS: [`select-from-table-with-auth-rls`](./supabase/functions/select-from-table-with-auth-rls/index.ts)
```

Comment:
```suggestion
- [`select-from-table-with-auth-rls`](./supabase/functions/select-from-table-with-auth-rls/index.ts): Retrieve data from an authenticated user via RLS
```

just a suggestion - once the list gets long it will be easier to skim down the page and group similar functions visually